src/sentry/search/snuba/backend.py

```diff
@@ -526,6 +526,12 @@ def _get_queryset_conditions(
             ),
         }
 
+        if features.has("projects:performance-issue-details-backend", projects[0]):
+            queryset_conditions.update(
+                {"category": QCallbackCondition(lambda category: Q(type__in=category))}
+            )
+            queryset_conditions.update({"type": QCallbackCondition(lambda type: Q(type=type))})
```

Comment:
Nit: Might make sense for the lambda params to be plural, since we're passing a list here